man/sd-id128.xml

```diff
@@ -267,6 +267,17 @@ int main(int argc, char **argv) {
 
   <xi:include href="libsystemd-pkgconfig.xml" />
 
+  <refsect1>
+    <title>History</title>
+    <para><function>sd_id128_equal</function> was added in version 252.</para>
```

Comment:
It would be nice to have the functions suffixed by `()` as done in other places in man pages.

Reply:
Fixed.